ui/packages/catalog-ui-search/src/main/webapp/component/query-settings/query-settings.view.js

```diff
@@ -269,23 +268,26 @@ module.exports = plugin(
     saveToModel() {
       this.model.set(this.toJSON())
     },
-    isValid() {
-      return this.settingsSortField.currentView.collection.models.length !== 0
+    validate() {
+      return {
+        isValid: this.settingsSortField.currentView.collection.models.length !== 0,
+        errorMessage: []
```

Comment:
Should there be an error message if the settings aren't valid?

:pencil2: If you base your `isValid` logic off the size of the `errorMessage` array, you could remove the isValid attribute. For example instead of checking `validation.isValid`, you could check `validation.errorMessages.length !== 0`